### DISCUSSION TITLE
Real-time suggestions in 0.4.0

### DISCUSSION THREAD

**Discussion by intitni** [intitni/CopilotForXcode#4] [General]
How do you feel about the real-time suggestions in version 0.4.0?

**Comment by ikerlostal** [comment]
I have followed all the steps but am not able to make the app run properly.

**Reply by ikerlostal** [reply]
Talking about real-time suggestions

**Reply by intitni** [reply]
Please make sure that Input Monitoring is enabled for Copilot for Xcode, and that you have run the command "Turn On Real-time Suggestions". If it's still not working, please run the command "Prefetch Suggestions" manually to see if there is any suggestion generated. If it does generate suggestions, maybe the app fails to find the project path, or the project path is incorrect, which makes the app think that real-time suggestions is turned off.

The current version 0.6.0 has a bug that the app will fail to provide real-time suggestions if the autocomplete panel from Xcode is displayed.

Some fixes will come in the next update.

**Reply by ChrisMiami** [reply]
Where does one find these commands to run them?
